client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/pivot/DateHistogramGroupSource.java

```diff
@@ -43,32 +46,157 @@
     private static final ParseField TIME_ZONE = new ParseField("time_zone");
     private static final ParseField FORMAT = new ParseField("format");
 
+    // From DateHistogramAggregationBuilder in core, transplanted and modified to a set
+    // so we don't need to import a dependency on the class
+    private static final Set<String> DATE_FIELD_UNITS;
```

Comment:
Why did you decide to put initialization in "static" block instead of initializing it with something like:
... Set DATE_FIELD_UNITS = new HashSet<>(Arrays.asList("year", ...));
?